aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_batch_unsampled_span_processor.py

```diff
@@ -0,0 +1,42 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+from typing import Optional
+
+from amazon.opentelemetry.distro._aws_attribute_keys import AWS_TRACE_FLAG_UNSAMPLED
+from opentelemetry.context import Context
+from opentelemetry.sdk.trace import ReadableSpan, Span
+from opentelemetry.sdk.trace.export import BatchSpanProcessor as BaseBatchSpanProcessor
+
+logger = logging.getLogger(__name__)
+
+SPANUNSAMPLED_FLAG = "OTEL_AWS_APP_SIGNALS_ENABLED"
+
+
+class BatchUnsampledSpanProcessor(BaseBatchSpanProcessor):
+
+    # pylint: disable=no-self-use
+    def on_start(self, span: Span, parent_context: Optional[Context] = None) -> None:
+        if not span.context.trace_flags.sampled:
+            span.set_attribute(AWS_TRACE_FLAG_UNSAMPLED, "True")
```

Comment:
should we use lower case `True` as attribute value which will be visible in Spans? It looks all other boolean attribute s are using lower case convention. Eg. `"aws.is.local.root":true,`